recipe/meta.yaml

```
@@ -1,52 +1,65 @@
+# celery.exceptions.SecurityError: Running a worker with superuser privileges
+{% set u_tests_to_skip = " test_check_privileges " %}
+# Failed: DID NOT RAISE any of (<class 'ValueError'>, <class 'Failed'>)
+{% set u_tests_to_skip = u_tests_to_skip + "or test_pytest_celery_marker_registration " %}
+# AssertionError: assert 'worker1 [2024-01-01 12:00:00]' in 'worker1 [2024-01-01 10:00:00] task received mytask(abc) args=(1,) kwargs={} \n'
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_task_received " %}
+# AssertionError: assert 'worker1 [2024-01-01 12:00:00]' in 'worker1 [2024-01-01 10:00:00] started: foo=bar\n'
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_non_task " %}
 
 package:
-    name: {{ name|lower }}
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: f4efebe6f8629b0da2b8e529424de376494f5b7a743c321c8a2ddc2b1414921c
-    patches:
-        - 0001-skip-a-test.patch
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c972ae7968c2b5281227f01c3a3f984037d21c5129d07bf3550cc2afc6b10a5
 
 build:
-    noarch: python
-    number: 0
-    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-    entry_points:
-        - celery = celery.__main__:main
+  skip: true  # [py<38]
+  entry_points:
+    - celery = celery.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
-    build:
+    host:
         - python
         - pip
-        - patch     # [not win]
-        - m2-patch  # [win]
-
+        - wheel >=0.33.1
+        - setuptools >=40.8.0
     run:
         - python
-        - click-didyoumean
-        - click-repl
-        - click-plugins
-        - pytz
-        - billiard>=3.6.3,<4.0
-        - kombu>=4.6.10,<4.7
-        - future >=0.18.0
-        - vine 1.3.0.*
-        - sqlalchemy
-        - cryptography
-    run_constrained:
-        - sphinx >=2.0
-
+        - billiard >=4.2.1,<5.0
+        - kombu >=5.5.2,<5.6
+        - vine >=5.1.0,<6.0
+        - click >=8.1.2,<9.0
+        - click-didyoumean >=0.3.0
+        - click-repl >=0.2.0
+        - click-plugins >=1.1.1
+        - backports.zoneinfo >=0.2.1  # [py<39]
+        - python-dateutil >=2.8.2
 test:
     requires:
-        - case >=1.3.1
-        - pytest >=4.5.0,<5.3.5
+        - pytest 8.4.1 # 8.3.5 version is required but we don't have this exact version, the newer one is compatible
         - pytest-subtests
-        - boto3 >1.9.178
-        - moto 1.3.7.*
+        - pytest-rerunfailures >=15.0 # [py39]
+        - dnspython
+        - google-cloud-storage
+        - azure-storage-blob>=12.15.0
+        - sqlalchemy
+        - pymongo
+        - boto3 >=1.26.143
+        - moto >=4.1.11,<5.1.0
+        - pip
     source_files:
         - t
```

```suggestion
        - t
        - pyproject.toml 
```
Because of these settings https://github.com/celery/celery/blob/v5.5.3/pyproject.toml#L1-L6

---

done

---

I tried to add this file and localy it even pass but it requires patch for pyproject.toml and shows errors on  CI like this:
`2025-08-13T07:14:42.11069+00:00   | INFO     |       for name in self._accessor.listdir(self):
2025-08-13T07:14:42.11226+00:00   | INFO     |   OSError: [Errno 24] Too many open files: '/private/var/folders/nz/j6p8yfhx1mv_0grj5xl4650h0000gp/T/pytest-of-builder/pytest-230'
2025-08-13T07:14:53.501034+00:00  | INFO     |   WARNING: Tests failed for celery-5.5.3-py310hca03da5_0.tar.bz2 - moving package to /var/folders/nz/j6p8yfhx1mv_0grj5xl4650h0000gp/T/abs_64xbbonyus/croot/broken
2025-08-13T07:14:53.583929+00:00  | INFO     |   
2025-08-13T07:14:53.585533+00:00  | INFO     |   CondaBuildUserError: TESTS FAILED: celery-5.5.3-py310hca03da5_0.tar.bz2`
Not sure that adding this file worth it.